data/transactions/logic/opcodes.go

```diff
@@ -311,8 +407,8 @@ var OpSpecs = []OpSpec{
 	{0x59, "extract_uint16", opExtract16Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
 	{0x5a, "extract_uint32", opExtract32Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
 	{0x5b, "extract_uint64", opExtract64Bits, asmDefault, disDefault, byteInt, oneInt, 5, modeAny, opDefault},
-	{0x5c, "base64_decode", opBase64Decode, asmBase64Decode, disDefault, oneBytes, oneBytes, fidoVersion, modeAny, field("e", &Base64Encodings).costs(25)},
-	{0x5d, "json_ref", opJSONRef, asmJSONRef, disDefault, twoBytes, oneAny, fidoVersion, modeAny, field("r", &JSONRefTypes)},
+	{0x5c, "base64_decode", opBase64Decode, asmDefault, disDefault, oneBytes, oneBytes, fidoVersion, modeAny, field("e", &Base64Encodings).costByLength(1, 1, 16)},
+	{0x5d, "json_ref", opJSONRef, asmDefault, disDefault, twoBytes, oneAny, fidoVersion, modeAny, field("r", &JSONRefTypes)},
```

Comment:
Yeah, totally reasonable to do that. I had not yet formed any real intent, but yes it would seem reasonable to do some benchmarks and use a linear size here.  I did it for base64 because the benchmarking was roughly in place.  I never really looked at how the cost was established for json_ref.

Reply:
@jannotti Got it - Let's not hold the PR on `json_ref` benchmarking.

I opened an issue to document the request:  https://github.com/algorand/go-algorand/issues/3860.

Resolving.